dockerfiles/Dockerfile.ci.os

```diff
@@ -0,0 +1,53 @@
+FROM debian:bookworm-slim AS base
+
+ENV ENVIRONMENT="local"
```

Comment:
These are now overwriting any input parameters. For example, If I pass -e APP_PORT=8082 the Dockerfile here will still expose the app on 8080. I believe you can remove the ENV calls in the Dockerfile.